cmd/kube-proxy/app/server_windows.go

```diff
@@ -178,31 +178,33 @@ func newProxyServer(config *proxyconfigapi.KubeProxyConfiguration, cleanupAndExi
 	}, nil
 }
 
+// getProxyMode attempts to use the kernel proxy (if requested), but will fall back to user space if necessary.  In cases where an unknown proxy is sent, userspace proxy is also used.
 func getProxyMode(proxyMode string, kcompat winkernel.KernelCompatTester) string {
 	if proxyMode == proxyModeUserspace {
 		return proxyModeUserspace
 	} else if proxyMode == proxyModeKernelspace {
-		return tryWinKernelSpaceProxy(kcompat)
+		proxyModeActual, err := tryWinKernelSpaceProxy(kcompat)
+		if err != nil {
+			fmt.Errorf("Can't determine whether to use windows kernel proxy %v (using proxy mode: %v)", err, proxyModeActual)
```

Comment:
This is just instnatiating an error but not doing anything with it. Was this supposed to be a `klog.Error` or `klog.Warning?

Reply:
OOPS ! :) i gotta fix this one

Reply:
Fxd